sdk/eventgrid/Azure.Messaging.EventGrid/tests/SessionRecords/EventGridClientLiveTests/CanPublishSingleEventAsync.json

```diff
@@ -29,7 +29,7 @@
     }
   ],
   "Variables": {
-    "DateTimeOffsetNow": "2021-03-05T16:18:36.8937492-08:00",
+    "DateTimeOffsetNow": "2021-03-06T00:18:36.8937492Z",
```

Comment:
This wasn't updated in the Autorest PR - I just happen to use Event Grid for testing Test Proxy.

Reply:
Why is this passing?

Reply:
It's not passing locally - not sure how CI passed.

Reply:
We need to investigate.